app/routes/users.routes.js

```diff
@@ -34,6 +39,12 @@ module.exports = function (app) {
 
     app.get('/users/authenticate', users.authenticateToken);
 
+    app.post('/users/from-token/last-name', users.updateLastName);
+
+    app.post('/users/from-token/profile-picture-url', users.updateProfilePictureUrl);
+
+    app.post('/users/from-token/profile-bio', users.updateProfileBio);
+
```

Comment:
I think these three post routes are still extra. They don't have anything to do with this PR, right? I think they should be removed.

Reply:
PR looking good otherwise!